Library/Formula/mplayer.rb

```diff
@@ -23,6 +23,9 @@ class Mplayer < Formula
     patch :DATA
   end
 
+  option "with-x11OSD", "Enable x11 for freefont/freetype menu support"
```

Comment:
done, tested, committed (to my fork) Thanks!

On 23 December 2015 at 18:19, Mike McQuaid <notifications@github.com> wrote:

> In Library/Formula/mplayer.rb
> <https://github.com/Homebrew/homebrew/pull/47287#discussion_r48328082>:
>
> > @@ -23,6 +23,9 @@ class Mplayer < Formula
> >      patch :DATA
> >    end
> >
> > +  option "with-x11OSD", "Enable x11 for freefont/freetype menu support"
>
> Just use with-x11 for the option name here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Homebrew/homebrew/pull/47287/files#r48328082>.
>